src/widget/woverview.cpp

```diff
@@ -434,50 +446,6 @@ void WOverview::paintEvent(QPaintEvent * /*unused*/) {
             const float offset = 1.0f;
             const float gain = static_cast<float>(length() - 2) / trackSamples;
 
-            // Draw range (loop)
```

Comment:
This code was moved after drawing the marks so the marks at the end of ranges don't get drawn over the duration text.